packages/gas-fee-controller/src/fetchBlockFeeHistory.ts

```diff
@@ -99,16 +99,16 @@ export type ExtractPercentileFrom<T> = T extends FeeHistoryBlock<infer P>[]
 const MAX_NUMBER_OF_BLOCKS_PER_ETH_FEE_HISTORY_CALL = 1024;
 
 /**
- * Uses `eth_feeHistory` (an EIP-1559 feature) to obtain information about gas fees from a range of
- * blocks that have occurred recently on a network.
+ * Uses `eth_feeHistory` (an EIP-1559 feature) to obtain information about gas
+ * fees from a range of blocks that have occurred recently on a network.
  *
  * To learn more, see these resources:
  *
- * - <https://infura.io/docs/ethereum#operation/eth_feeHistory>
- * - <https://github.com/zsfelfoldi/feehistory/blob/main/docs/feeHistory.md>
- * - <https://github.com/ethereum/go-ethereum/blob/57a3fab8a75eeb9c2f4fab770b73b51b9fe672c5/eth/gasprice/feehistory.go#L180>
- * - <https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md>
- * - <https://gas-api.metaswap.codefi.network/testFeeHistory>
+ * - <https://infura.io/docs/ethereum#operation/eth_feeHistory>.
```

Comment:
Hmm, seems like we shouldn't be putting periods at the ends of these list items. 

I swear there was an issue open for this on `eslint-plugin-jsdoc` but I can't seem to find it now...

Reply:
The links are still clickable as normal, so I don't see the issue with it?

Reply:
They are, that's true. It's just that each line is not a sentence, so periods look kind of extra. But I guess it's fine for now until we figure out a way around this.